search/includes/classes/class-versioning.php

```diff
@@ -764,13 +778,26 @@ public function filter__pre_ep_index_sync_queue( $bail, $sync_manager, $indexabl
 			return $bail;
 		}
 
+		$queue = \Automattic\VIP\Search\Search::instance()->queue;
+
 		foreach ( $inactive_versions as $version ) {
-			foreach ( $sync_manager->sync_queue as $object_id => $value ) {
-				$options = array(
-					'index_version' => $version['number'],
-				);
+			$options = array(
+				'index_version' => $version['number'],
+			);
 
-				\Automattic\VIP\Search\Search::instance()->queue->queue_object( $object_id, $indexable_slug, $options );
+			foreach ( $sync_manager->sync_queue as $object_id => $value ) {
+				/**
+				 * Filter do determine the priority of the index synchronization job
+				 *
+				 * @param int $priority         Priority
+				 * @param int $object_id        Object ID
+				 * @param string $object_type   Object type
+				 * @return int                  Priority
+				 */
+				$priority            = apply_filters( 'ep_index_sync_priority', self::INDEX_SYNC_JOB_DEFAULT_PRIORITY, $object_id, $indexable_slug );
```

Comment:
I think we should use the INACTIVE_VERSION default priority. I don't think we need to differentiate in inactive versions too much. So using `INACTIVE_VERSION_JOB_DEFAULT_PRIORITY` is fine here I think.

Which would mean using `vip_inactive_version_reindex_priority` filter probably

Reply:
Fixed in 09af12f04